osf/external/chronos.py

```diff
@@ -69,17 +69,19 @@ def serialize_manuscript(cls, journal_id, preprint, status=ChronosSubmissionStat
             'PROVIDER_MANUSCRIPT_ID': preprint._id,
             'CHRONOS_JOURNAL_ID': journal_id,
             'MANUSCRIPT_URL': preprint.url,
+            'KEYWORDS': ','.join(preprint.tags.all().values_list('name', flat=True)),
+            'EXTRAVALUES': 'Provider:{}'.format(preprint.provider.name),
```

Comment:
It's unclear to me whether this should be `EXTRAVALUES`, `EXTRA_VALUES`, or `ExtraValues` (as they told us).

Reply:
Chronos responded and clarified. The payload is changed accordingly.